library/include/rocblas-functions.h

```diff
@@ -1406,6 +1406,122 @@ ROCBLAS_EXPORT rocblas_status rocblas_izamax(rocblas_handle                handl
+         or the sum of magnitude of the real and imaginary parts of elements if x is a complex
+
+    @param[in]
+    handle    rocblas_handle.
+              handle to the rocblas library context queue.
+    @param[in]
+    n         rocblas_int
+              number of elements in each vector x_i
+    @param[in]
+    x         array of pointers storing the different vector x_i on the GPU.
+    @param[in]
+    incx      rocblas_int
+              specifies the increment for the elements of each x_i. incx must be > 0.
+    @param[in]
+    batch_count rocblas_int
+              number of instances in the batch, must be > 0.
+    @param[out]
+    result
+              pointers to array of batch_count size for results. either on the host CPU or device GPU.
+              return is 0 if n, incx<=0.
+    ********************************************************************/
+
+ROCBLAS_EXPORT rocblas_status rocblas_isamax_batched(rocblas_handle     handle,
+                                                     rocblas_int        n,
+                                                     const float* const x[],
+                                                     rocblas_int        incx,
+                                                     rocblas_int        batch_count,
+                                                     rocblas_int*       result);
+
+ROCBLAS_EXPORT rocblas_status rocblas_idamax_batched(rocblas_handle      handle,
+                                                     rocblas_int         n,
+                                                     const double* const x[],
+                                                     rocblas_int         incx,
+                                                     rocblas_int         batch_count,
+                                                     rocblas_int*        result);
+
+ROCBLAS_EXPORT rocblas_status rocblas_icamax_batched(rocblas_handle                     handle,
+                                                     rocblas_int                        n,
+                                                     const rocblas_float_complex* const x[],
+                                                     rocblas_int                        incx,
+                                                     rocblas_int                        batch_count,
+                                                     rocblas_int*                       result);
+
+ROCBLAS_EXPORT rocblas_status rocblas_izamax_batched(rocblas_handle                      handle,
+                                                     rocblas_int                         n,
+                                                     const rocblas_double_complex* const x[],
+                                                     rocblas_int                         incx,
+                                                     rocblas_int  batch_count,
```

Comment:
clang format?

Reply:
mystery

Reply:
> 
> 
> clang format?

> mystery

Does the `.githooks/pre-commit` script get run when you commit your changes? It requires the latest ROCm to be installed, for it to get `clang-format`.

Are you using a MS-DOS or Mac editor? The `.githooks/pre-commit` script converts CR-LF to LF for line endings, and removes trailing spaces on the lines. I did not anticipate LF-CR, so I did not plan for the script to standardize it to LF.

You should make sure your environment runs `.githooks/pre-commit` correctly. On Linux, it "just works".

Reply:
Using rack72 with a linux OS ... but the clang-format checking on the CI passed!..

Reply:
Thanks, I don't think the hook is running ... I already had clang format failures with the CI. Hence, I have my own command line to loop over and applying the clang format. 
But again, the CI passed on the format.